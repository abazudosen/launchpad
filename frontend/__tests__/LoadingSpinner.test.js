/* @flow */

import React from 'react';
import { LoadingSpinner } from '../LoadingSpinner';
import renderer from 'react-test-renderer';

describe('Loading Spinner', () => {
  test('renders normal', () => {
    const component = renderer.create(
      <LoadingSpinner size="small" className="test" />,
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
