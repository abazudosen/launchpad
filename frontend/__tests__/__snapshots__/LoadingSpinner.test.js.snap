// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Spinner renders normal 1`] = `
<div
  className="loading-spinner small test"
>
  <div
    className="spinner-wheel"
  />
  <div
    className="spinner-ticks"
  >
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
    <div
      className="spinner-tick"
    />
  </div>
</div>
`;
